root/edit/components/EditSidebar.js

```diff
@@ -96,7 +96,7 @@ const EditSidebar = ({
       <ul className="links">
         <li><a href="/doc/Introduction_to_Voting">{l('Voting FAQ')}</a></li>
         <li><a href="/doc/Editing_FAQ">{l('Editing FAQ')}</a></li>
-        <li><a href="/doc/Edit_Types">{l('Edit Types')}</a></li>
+        <li><a href="/doc/Edit_Types">{l('Edit types')}</a></li>
```

Comment:
It could be mistaken with the verb form:

```suggestion
        <li><a href="/doc/Edit_Types">{l('Edit types', 'noun')}</a></li>
```